packages/frontend/components/Image/Image.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+export interface ImageInterface {
+  id: string
+  creator: string
+  url: string
+}
+
+interface ImageProps {
+  key: string
+  image: ImageInterface
+}
+export default function Image({ image }: ImageProps) {
+  return (
+    <div>
+      <p>Creator: {image.creator}</p>
+      <img src={image.url} width={300} height={250}></img>
```

Comment:
Height and width should be in a seperate scss file. Also you can make img self closing
```suggestion
      <img src={image.url} width={300} height={250} />
```

Reply:
fixed